packages/core/src/reducer.js

```diff
@@ -76,6 +77,13 @@ const reducer = handleActions({
       }
     );
   },
+  [ACTION_DISPOSE]: (state, action) => {
+    const apiName = getName(action);
+    if (state.hasOwnProperty(apiName)) {
+      delete state[apiName];
```

Comment:
by deleting, I didn't mean using `delete` keyword. Due to the fact that `delete` is mutative, this line will break redux assumption of immutability.